CMakeLists.txt

```diff
@@ -59,11 +59,22 @@ common_find_package(LibJpegTurbo)
 common_find_package(Rockets)
 if(ROCKETS_FOUND AND ROCKETS_USE_LIBWEBSOCKETS)
   option(BRAYNS_NETWORKING_ENABLED "Activate networking interfaces" ON)
-  if(BRAYNS_NETWORKING_ENABLED)
+else()
+  option(BRAYNS_NETWORKING_ENABLED "Activate networking interfaces" OFF)
+endif()
+
+if(BRAYNS_NETWORKING_ENABLED)
+  # We require turbojpeg for networking to get image streaming working
+  common_find_package(LibJpegTurbo REQUIRED)
+  if(ROCKETS_FOUND AND ROCKETS_USE_LIBWEBSOCKETS)
```

Comment:
why not for consistency with other options do `common_find_package(Rockets REQUIRED)`? If it's enabled, it shall error imo, same if `ROCKETS_USE_LIBWEBSOCKETS` is not defined

Reply:
Done